tests/test_interface_switchport.rb

```diff
@@ -100,6 +100,8 @@ def test_switchport_vtp_disabled_unsupported_mode_fex
   end
 
   def test_switchport_vtp_enabled_access
+    skip('Cannot apply VTP config because port mode is access') if
+      node.product_id =~ /N(5|6|7)/
```

Comment:
This message is a little unclear. Consider:

`'Platform does not support VTP when switchport mode is access'`

Also, consider putting these in a helper method, a la:

```
def platform_supports_vtp_on_mode_access
  skip(`'Platform does not support VTP when switchport mode is access') if
    node.product_id =~ /N(5|6|7)/
 end
```